packages/datastore/src/datastore.ts

```diff
@@ -92,27 +131,91 @@ class Datastore implements IIterable<Table> {
    * `O(log32 n)`
    */
   get<S extends Schema>(schema: S): Table<S> {
-    throw '';
+    return this._tables[schema.id] as Table<S>;
   }
 
   /**
+   * Begin a new transaction in the store.
+   *
+   * @returns The id of the new transaction
    *
+   * #### Notes
+   * This will allow the state of the store to be mutated
+   * thorugh the `update` method on the individual tables.
+   *
+   * After the updates are completed, `endTransaction` should
+   * be called.
    */
   beginTransaction(): string {
```

Comment:
It's possible  that the user forgets to call `endTransaction`. If you make the datastore a message handler, you can post a conflatable message to yourself when the transaction begins. When you receive that message, you can check to see if the transaction was not closed, and close it automatically.